src/configs.js

```diff
@@ -118,6 +118,16 @@ module.exports = {
 				required: true,
 				isVisible: ({ feedback, interval }) => feedback === true && interval === 0,
 			},
+			{
+				type: 'number',
+				id: 'maxClips',
+				label: 'Maximum number of clips',
+				width: 6,
+				min: 1,
+				max: 100,
+				default: 5,
+				required: true,
+			},
```

Comment:
The module should be able to handle however many clips are available in PlayoutBee and should dynamically create the necessary variables it wants to expose. I'm not keep on setting (or having the user set) some arbitrary limit.

Reply:
Intention about the config value was, that one could predefine the various buttons even if clips are not yet there. E.g. you have 3 clips within playoutbee, but you want to configure 10 buttons in companion and after you add clip 4 to playoutbee it automatically shows up in companion without doing anything. But I get your point and lets take this point up in a separate discussion, I will open an issue for that.